examples/gatsbygram/package.json

```diff
@@ -42,16 +42,19 @@
   "main": "n/a",
   "scripts": {
     "lint": "./node_modules/.bin/eslint --ext .js,.jsx --ignore-pattern public .",
+    "pretest": "gatsby-dev --set-path-to-repo ../../ && gatsby-dev --scan-once --quiet && npm run fix-permissions",
     "test": "start-server-and-test develop http://localhost:8000 cy:run",
     "develop": "gatsby develop",
     "dev": "gatsby develop",
     "build": "gatsby build",
     "deploy": "gatsby build --prefix-paths && gh-pages -d public",
+    "fix-permissions": "chmod +x node_modules/.bin/gatsby node_modules/.bin/gatsby-dev",
```

Comment:
Janky. Anyone seen this before? Basically running the `pretest` script _once_ breaks the executables `gatsby` and `gatsby-dev` in `node_modules/.bin`